src/common/store/constants/index.js

```diff
@@ -0,0 +1,10 @@
+/*
+ * Export named constants only
+ * e.g.
+ * export const MY_ACTION_CONST = MY_ACTION_CONST;
+ * export const MY_ACTION_2_CONST = MY_ACTION_2_CONST;
```

Comment:
I know this is nitpicky, but the right side of these two lines should be strings. So the lines should read:

```js
/* ...
 * export const MY_ACTION_CONST = 'MY_ACTION_CONST';
 * export const MY_ACTION_2_CONST = 'MY_ACTION_2_CONST';
 * ...
 */
```